src/templates/labels_form_inner_template.html

```diff
@@ -42,6 +42,15 @@
                     </li>
                     <li class="add_time_label">
                         <a href="#" data-label="[time-hrs:mins]">Time (hrs:mins)</a>
+                    </li>
+					<li class="add_depth_label"  title="Add a label that shows the current Z-index">
+                        <a href="#" data-label="[depth-index]">Depth (Z-index)</a>
+                    </li>
+                    <li class="add_depth_label"  title="Add a label that shows the current depth">
+                        <a href="#" data-label="[depth-unit]">Depth (unit)</a>
+                    </li>
+					<li class="add_roi_label"  title="Add a label that shows the viewport X and Y">
+                        <a href="#" data-label="X: [roi-x] Y: [roi-y]">X and Y (pixel)</a>
```

Comment:
I think using `roi` here could be confusing since there's no ROI on the image (even more confusing if there was an ROI). Maybe use e.g. `[viewport-x]` or if that's too long then `[view-x]`?

Reply:
Viewport/view sounds better than roi. We could also make aliases, to have one explicit name and one faster to type:
-> viewport : view : v
->time : t
->depth : d
->channels : c
-> name : n